Documentation/subcommands/dependency.md

```diff
@@ -0,0 +1,49 @@
+# acbuild dependency
+
+Dependencies are ACIs separate from the current ACI, that are placed down into
+the rootfs before the files from the current ACI.
+
+The ordering of an ACI's dependencies is significant. Let's say the current ACI
+has two dependencies, A and B, where A comes before B. If both A and B contain
+the file `/bin/sh`, and the current ACI does not have this file, then the
+`/bin/sh` from B is what will appear in the container when it is run.
+
+At the time of writing acbuild simply puts the dependency in the manifest in
+the order they were added. If the ordering of an ACI's dependencies is
+incorrect, the user must remove each dependency manually and then add them in
+the correct order.
```

Comment:
Is there a way to list the current dependencies or to remove all dependencies if the user forgets the dependencies? The current way seems to be:

```
json_pp < .acbuild/currentaci/manifest
```